test/cli/cli.spec.ts

```diff
@@ -0,0 +1,46 @@
+import tape from 'tape'
+import { spawn } from 'child_process'
+
+tape('[CLI]', (t) => {
+  t.test('should start and begin downloading blocks', { timeout: 160000 }, (t) => {
+    t.plan(1)
+    const file = require.resolve('../../dist/bin/cli.js')
+    const child = spawn(process.execPath, [file])
+
+    const timeout = setTimeout(() => {
+      child.kill('SIGINT')
+    }, 120000)
+
+    const end = () => {
+      clearTimeout(timeout)
+      child.kill('SIGINT')
+      t.end()
+    }
+
+    child.stdout.on('data', (data) => {
+      const message = data.toString()
+      if (message.toLowerCase().includes('error')) {
+        t.fail(message)
+        end()
+      }
+      if (message.includes('Imported blocks')) {
```

Comment:
FYI If data chunk size exceeds 16Kb (default value for [readableHighWaterMark](https://nodejs.org/dist/latest-v14.x/docs/api/stream.html#stream_readable_readablehighwatermark)), then it will be split into several chunks to fit 16Kb size limit. So it's possible to meet the situation, when some part of 'Imported block' is in the first chunk and other is in another. It could be highly possible if script writes big bunch of data in one libuv event loop cycle, not sure if Client will ever do it. But if it will, then there will be false negative tests.

Reply:
ah good to know, thanks, I was wondering about how it may be chunked. Any suggestions on a better or more reliable way to parse the incoming data?

Reply:
I think native IPC could be used here, it will simplify things dramatically. I've added #177 with a support of IPC logger. So we can just merge this PR (cause it works) and then apply that PR. Or if #177 wouldn't be applied to add IPC logger into current code.